count_calls/count_calls.py

```diff
@@ -32,8 +32,7 @@ def main(argv):
         # All good.
         print('The recording hit "{}" {} time(s).'.format(
             func_name,
-            res.result_data['hit-count'],
-            ))
+            res.result_data['hit-count'],))
```

Comment:
Hm, why? I don't think this is correct according to our style.

Reply:
I changed it because the linter did not like my previous commit:
== PYCODESTYLE == 
count_calls/count_calls.py:36:13: E123 closing bracket does not match indentation of opening bracket's line
count_calls/count_calls.py:51:17: E123 closing bracket does not match indentation of opening bracket's line

Reply:
The linter was wrong, I have undone the change.